modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordDropDown.kt

```diff
@@ -33,6 +34,30 @@ class MirrordDropDown : ComboBoxAction(), DumbAware {
         }
     }
 
+    private class ToggleEnabledByDefault : AnAction("Enable/ Disable mirrord by Default") {
+        override fun actionPerformed(e: AnActionEvent) {
+            val newValue = !MirrordSettingsState.instance.mirrordState.enabledByDefault
+            MirrordSettingsState.instance.mirrordState.enabledByDefault = newValue
+            // popup containing new value of enabledByDefault
+            val service = e.project?.service<MirrordProjectService>() ?: return
+            val notification = if (newValue) {
+                service.notifier.notification(
+                    "mirrord set to be enabled by default",
+                    NotificationType.IDE_UPDATE
+                )
+            } else {
+                service.notifier.notification(
+                    "mirrord set to be disabled by default",
+                    NotificationType.IDE_UPDATE
+                )
+            }
+
+            notification.fire()
+
+            return
+        }
+    }
```

Comment:
No need for this, I think. The user can change this in the settings

Reply:
no item in the dropdown menu at all?

Reply:
Yup